src/pages/Create/index.js

```diff
@@ -46,6 +49,13 @@ class Create extends Component {
         }); 
     }
 
+    newPath = () => {
```

Comment:
What is this suppose to mean? newPath ? It's not very describing. Please give your functions good names. Let them be short, but be enough to describe themself and what they do.